2-Browsers/Week1/homework/ex1-bookList/index.js

```diff
@@ -19,7 +19,35 @@
 //cspell: enable
 
 function createBookList(books) {
-  // your code goes in here, return the ul element
+  const ulBook = document.createElement('ul');
+
+  books.forEach((book) => {
```

Comment:
Nice use of `.forEach`. I personally prefer `for (const ... of ...)` nowadays, but this isn't wrong at all.

Reply:
Thanks for this point

Do you advice me to use for (const ... of ...) instead  of .forEach to loop through an array in General

Reply:
Yes, I would. But it's a preference.